analysis/stat_tests.py

```diff
@@ -48,51 +51,54 @@ def test_pair(measurements_x, measurements_y):
     for f_i in fuzzers:
         row = []
         for f_j in fuzzers:
-            if f_i == f_j:
-                # TODO(lszekeres): With Pandas 1.0.0+, switch to:
-                # p_value = pd.NA
-                p_value = np.nan
-            elif set(samples[f_i]) == set(samples[f_j]):
-                p_value = np.nan
-            else:
-                p_value = test_pair(samples[f_i], samples[f_j])
-            row.append(p_value)
+            value = np.nan
+            if f_i != f_j and set(samples[f_i]) != set(samples[f_j]):
+                res = test_pair(samples[f_i], samples[f_j])
+                value = getattr(res, statistic)
+            row.append(value)
         data.append(row)
 
-    p_values = pd.DataFrame(data, index=fuzzers, columns=fuzzers)
-    return p_values
+    return pd.DataFrame(data, index=fuzzers, columns=fuzzers)
 
 
 def one_sided_u_test(benchmark_snapshot_df, key):
     """Returns p-value table for one-tailed Mann-Whitney U test."""
-    return _create_p_value_table(benchmark_snapshot_df,
-                                 key,
-                                 ss.mannwhitneyu,
-                                 alternative='greater')
+    return _create_pairwise_table(benchmark_snapshot_df,
+                                  key,
+                                  ss.mannwhitneyu,
```

Comment:
here and below, would it make sense to pass `lambda xs, ys, alt: ss.mannwhitneyu(xs, ys, alt).pvalue` instead of `ss.mannwhitneyu` in order to remove the `statistic='pvalue'` parameter of `_create_pairwise_table`?

Reply:
Altered the arguments for these four functions and removed the 'alternative' and 'statistic' arguments.

Reply:
Thanks!